packages/@aws-cdk/aws-cognito-identitypool-alpha/lib/identitypool.ts

```diff
@@ -1,88 +1,64 @@
-import {
-  CfnIdentityPool,
-  IUserPool,
-  IUserPoolClient,
-} from 'aws-cdk-lib/aws-cognito';
-import {
-  IOpenIdConnectProvider,
-  ISamlProvider,
-  Role,
-  FederatedPrincipal,
-  IRole,
-} from 'aws-cdk-lib/aws-iam';
-import {
-  Resource,
-  IResource,
-  Stack,
-  ArnFormat,
-  Lazy,
-  Token,
-} from 'aws-cdk-lib/core';
-import {
-  Construct,
-} from 'constructs';
-import {
-  IdentityPoolRoleAttachment,
-  IdentityPoolRoleMapping,
-} from './identitypool-role-attachment';
-import {
-  IUserPoolAuthenticationProvider,
-} from './identitypool-user-pool-authentication-provider';
+import { CfnIdentityPool, IUserPool, IUserPoolClient } from 'aws-cdk-lib/aws-cognito';
+import { IOpenIdConnectProvider, ISamlProvider, Role, FederatedPrincipal, IRole } from 'aws-cdk-lib/aws-iam';
+import { Resource, IResource, Stack, ArnFormat, Lazy, Token } from 'aws-cdk-lib/core';
+import { Construct } from 'constructs';
+import { IdentityPoolRoleAttachment, IdentityPoolRoleMapping } from './identitypool-role-attachment';
+import { IUserPoolAuthenticationProvider } from './identitypool-user-pool-authentication-provider';
 
 /**
- * Represents a Cognito IdentityPool
+ * Represents a Cognito identity pool
  */
 export interface IIdentityPool extends IResource {
   /**
-   * The id of the Identity Pool in the format REGION:GUID
```

Comment:
Anything that refers to an actual resource, like an Identity Pool, should be capitalized.

Reply:
Thanks for clearing that up, I wasn't 100% on what the proper formatting should be for the docstrings. Will fix it up